specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -21896,6 +21899,14 @@
           {
             "$ref": "#/parameters/slotNameParameter"
           },
+          {
+            "name": "configurationName",
+            "in": "path",
+            "description": "Name of workflow configurations.",
+            "required": true,
+            "type": "string",
+            "pattern": "^connections$"
```

Comment:
Usually if only one value is allowed, the parameter would be defined as an x-ms-enum (either modelAsString=false if never change, or modelAsString=true for future extension).

Reply:
Also if this is not API added in this new api-version, changing the path would break SDK.